descriptors/ModuleDescriptor-template.json

```diff
@@ -1079,6 +1079,30 @@
         }
       ]
     },
+    {
+      "id": "orders.bind-pieces",
+      "version": "1.0",
+      "handlers": [
+        {
+          "methods": ["POST"],
+          "pathPattern": "/orders/bind-pieces",
+          "permissionsRequired": [
+            "orders.bind-pieces.collection.post"
+          ],
+          "modulePermissions": [
+            "orders-storage.pieces.collection.get",
+            "orders-storage.pieces.item.put",
+            "orders-storage.po-lines.collection.get",
+            "orders-storage.po-lines.item.put",
+            "orders-storage.purchase-orders.item.get",
+            "orders-storage.purchase-orders.item.put",
+            "orders-storage.titles.collection.get",
```

Comment:
there is no permission to create item, also we fetch po line by id, so need orders-storage.po-lines.item.get instead of orders-storage.po-lines.collection.get.
Also we save title with bind item ids, so need permission to update title as well

Reply:
added module permissions